src/Sylius/Behat/Context/Transform/PromotionContext.php

```diff
@@ -52,4 +57,21 @@ public function getPromotionByName($promotionName)
 
         return $promotion;
     }
+
+    /**
+     * @Transform /^coupon "([^"]+)"$/
+     * @Transform /^"([^"]+)" coupon$/
+     * @Transform :coupon
+     */
+    public function getPromotionCouponByCode($promotionCouponCode)
```

Comment:
Maybe separate context for coupons?

Reply:
We might get back to it after stable release.